tests/Command/MakeMigrationCommandTest.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Symfony\Bundle\MakerBundle\Tests\Command;
+
+use Symfony\Bundle\MakerBundle\Command\MakeMigrationCommand;
+use PHPUnit\Framework\TestCase;
+
+class MakeMigrationCommandTest extends TestCase
+{
+
+}
```

Comment:
We should add a test here: https://github.com/symfony/maker-bundle/blob/master/tests/Command/FunctionalTest.php#L68

And then we might not need a separate test at all :)